### ISSUE TITLE
Enable a request params disable

### ISSUE THREAD

**Issue by clement-hvt** [apideck-libraries/portman#484]
Options exist to disable request parameter but not the other way around. Am I missing something or is this not planned?
To avoid the risk of breaking changes we could add an `enable` option.

**Comment by thim81** [comment]
hi @clement-hvt 

 Do you want to enable request parameters?

Through https://github.com/apideck-libraries/portman#portman---overwrites-properties you should be able to this via `disable: false`. They are enabled by default normally.
If you want to add request parameters, you can use `insert: true`

**Comment by clement-hvt** [comment]
@thim81,

if in my OpenAPI doc my parameter has `required` set to false, the parameter is disable in Postman for all requests who use it and that's normal. But,  if I want to activate it for a request neither:

```json
"overwriteRequestQueryParams": [
  {
    "key": "include[merchant]",
    "value": "xx",
    "disable": false
  }
]
```

or 

```json
"overwriteRequestQueryParams": [
  {
    "key": "include[merchant]",
    "value": "xx",
    "insert": true
  }
]
```

enable the parameter.
The overwrite is located in a test variation.

**Comment by thim81** [comment]
We will investigate this and fix it if it would be a bug.

Thanks for the detailed example.

**Comment by magicDGS** [comment]
I also found this issue with my configuration:

```yaml
...
variationTests:
    - openApiOperation: 'GET::/cardinfo.php'
      openApiResponse: '200'
      variations:
        - name: 'Misc parameter'
          overwrites:
            - overwriteRequestQueryParams:
              - key: misc
                value: 'yes'
                overwrite: true
                disable: false
...
```

Generates a postman collection where the parameter is:

```json
"query": [
  ...
  {
    "disabled": true,
    "description": {
      "content": "Show additional response info (Card Views, Beta Name, etc.).",
      "type": "text/plain"
    },
    "key": "misc",
    "value": "yes"
  },
  ...
]
```

**Comment by magicDGS** [comment]
I think that I found the issue, @thim81:

https://github.com/apideck-libraries/portman/blob/2c4ccf93ee40286024fc22103a130af112b54169/src/application/overwrites/overwriteRequestQueryParams.ts#L45-L50

That code only checks for `disable=true` but if it is `disable=false` it does not overwrites the state explicitly. Can you please create a fix and a release with it? This bug is preventing me to define tests for an API where most of the query parameters are optional and thus disabled by default!

**Comment by thim81** [comment]
hi @magicDGS 

We have a [fix](https://github.com/apideck-libraries/portman/pull/528) in the making for this issue

**Comment by thim81** [comment]
hi @magicDGS 

We have just released version v1.23.1 of Portman, which allows you to mark Postman query paramaters as `disbled=false` by using the `overwriteRequestQueryParams` with the config `disable: false`.

I'll close this issue, in case the issue is still persisting, please open a new issue.

**Comment by magicDGS** [comment]
Thanks for the feedback, @thim81! I will try as soon as I have time to use it on my project.
